includes/block-migrate/class-coblocks-block-migration.php

```diff
@@ -111,7 +110,11 @@ protected function block_wrapper() {
 	 *
 	 * @return string attribute value.
 	 */
-	protected function get_attribute_from_classname( $classname_prefix, DOMElement $element ) {
+	protected function get_attribute_from_classname( $classname_prefix, $element ) {
+		if ( empty( $element ) ) {
+			return '';
+		}
+
```

Comment:
What is the reason for removing the `DOMElement` type hint? We don't want to accept any value here since we expect the attributes property to exist with the `getNamedItem` method.

Reply:
Okay reverted this change. I think this was something that I changed early on while I was testing different approaches but this is not needed now.